src/modules/annotations.js

```diff
@@ -363,7 +363,8 @@ import { jsPDF } from "../jspdf.js";
     //TODO We really need the text baseline height to do this correctly.
     // Or ability to draw text on top, bottom, center, or baseline.
     y += height * 0.2;
-    this.link(x, y - height, width, height, options);
+    x += width * 0.5;
+    this.link(x - width, y - height, width, height, options);
```

Comment:
I think we should do this conditionally depending on the `options.align` and `options.maxWidth` properties.

Reply:
Hey @HackbrettXXX  I guess you are right, checked it based on align condition. But due to some reason build got interrupted and ci throws error. Can you do a re-run or help me out on that part.